MovieApp/MovieApp/App/CoreUI/Extensions/UserDefaults+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension UserDefaults {
+    
+    @objc dynamic var favorites: [Int] {
```

Comment:
Why do you need this?

Reply:
For cleaner code in [FavoritesUserDefaultsDataSource](https://github.com/srcxp/MovieApp/blob/feature/reactive-networking/MovieApp/MovieApp/App/Features/Movies/DataSources/UserDefaultsDataSource/FavoritesUserDefaultsDataSource.swift#L18) where I use \.favorites. In case of more sections stored in UserDefaults I believe an Extension like this would make code easier to read.